frontend/src/scenes/persons/personsLogic.ts

```diff
@@ -52,6 +60,34 @@ export const personsLogic = kea<personsLogicType<PersonPaginatedResponse>>({
                 },
             },
         ],
+        person: [
+            null as PersonType | null,
+            {
+                loadPerson: async (id) => {
+                    const response = await api.get(`api/person/?distinct_id=${id}`)
+                    if (!response.results.length) {
+                        router.actions.push('/404')
+                    }
+                    return response.results[0]
+                },
+                setPerson: (person: PersonType) => {
+                    // Used after merging persons to update the view without an additional request
+                    return person
+                },
+            },
+        ],
+        deletedPerson: [
+            false,
+            {
+                deletePerson: async () => {
+                    if (!values.person) {
+                        return
+                    }
+                    await api.delete(`api/person/${values.person.id}`)
+                    return true
```

Comment:
Is this read anywhere?

Reply:
No, but it seems that if we don't return a value the reducer raises an exception and the action lifecycle is not completed.

Reply:
Aha. That's because this is marked as a loader - even though it isn't loading any data that's being used.

I think this should be a listener: https://kea.js.org/docs/guide/concepts#listeners